Doc/library/importlib.rst

```diff
@@ -529,7 +529,8 @@ ABC hierarchy::
     .. abstractmethod:: is_resource(name)
 
         Returns ``True`` if the named *name* is considered a resource.
-        :exc:`FileNotFoundError` is raised if *name* does not exist.
+
+        The abstract method always returns ``False``.
```

Comment:
Why change the behavior here? Could you make an issue on https://bugs.python.org/ describing the rationale?

Reply:
Ok, I'll try